components/webgpu/canvas_context.rs

```diff
@@ -557,7 +557,7 @@ impl crate::WGPU {
             else {
                 warn!("Failure obtaining available staging buffer");
                 sender
-                    .send(Snapshot::cleared(configuration.size).as_ipc())
+                    .send(Snapshot::cleared(configuration.size).to_shared())
```

Comment:
Here we alloc vec, then we alloc IpcSharedMemory. I think we should add `SharedSnapshot::cleared` to avoid vec alloc.